clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -3535,6 +3576,12 @@ void handleFailure(Throwable throwable) {
         return new ListPartitionReassignmentsResult(partitionReassignmentsFuture);
     }
 
+    private long calculateNextAllowedRetryMs(long currNextAllowedRetryMs) {
+        return (currNextAllowedRetryMs == 0) ?
+            time.milliseconds() + retryBackoffMs :
+            currNextAllowedRetryMs + retryBackoffMs;
```

Comment:
To keep this logic same as one in `Call::fail` method, lets set the new time as:

> nextAllowedTryMs = now + retryBackoffMs

Reply:
Done.